### PR TITLE
AB-356: Redirects MBID to it's original entity using musicbrainz data for recording entity

### PR DESCRIPTION
**Pull request by rsh7**
An integration to actually work with musicbrainz data over faster access. This PR consists of a script which takes mbids as input and results in redirecting the MBID of an entity to it's original entity.

### CODE REVIEW

File: webserver/external/get_entities.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+              FROM musicbrainz.recording_gid_redirect
+        """)
+        result = connection.execute(query)
+        mbids = result.fetchall()
+
+        recording_mbids = []
+        for mbid in mbids:
+            recording_mbids.append(str(mbid[0]))
+
+        gids_with_redirected_ids = get_original_entity(recording_mbids)
```
[the remark this answers is not in the document]
Comment (rsh7) [reply webserver/external/get_entities.py]: The part of `MBID` redirect is to use the information to determine when two distinct MBIDs in the database refer to the same `recording`.
So, I am storing the result in the form of dictionary with `keys` as `MBIDs` and `values` as `ids` of the recording to which it is redirected to.
And we can use this information to implement any functionality in future and for now we would use it for evaluating 2 methods of db access

[the remark this answers is not in the document]
Comment (paramsingh) [reply webserver/external/get_entities.py]: I understand that, but the function should still return `gids_with_redirected_ids`

**Comment by rsh7** [comment]
Rebased this with the base branch. :+1:

File: webserver/external/get_entities.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (rsh7) [reply webserver/external/get_entities.py]: Done!

**Comment by rsh7** [comment]
@paramsingh, error in function `dump.import_db_dump(path)`
file - db/test/test_dump.py
It is related to some of your recent PRs I guess.

**Comment by paramsingh** [comment]
@brainzbot retest this please

**Pull request merged**